File: Lista 6/L6_Q2.c
#include <stdio.h>

int main(){
	float v, d;
	
	printf("Qual o valor da compra: ");
	scanf("%f", &v);
	
	if(v >= 100){
		printf("Qual o valor do desconto em porcentagem: ");
		scanf("%f", &d);

		d = d / 100;
		v -= v * d;

		printf("eh possivel conceder o desconto\nValor a ser pago eh R$%.2f", v);
	}else
		printf("Nao eh possivel conceder o desconto");
	
	return 0;
}

File: 1_Período/Lista 6/Python/L6_Q2.py
valor = float(input("\nInforme o valor da compra: "))
desconto = float(input("Informe o percentual de desconto: "))

valor_com_desconto = valor - valor * desconto/100

if valor_com_desconto < 100:
    print(f"\nNão é possivel conceder o desconto T.T\nO valor da compra é de R${valor}\n")
else:
    print(f"\nÉ possivel conceder o desconto!!\nO valor da compra é de R${valor_com_desconto}\n")